test/Stateless.Tests/DotGraphFixture.cs

```diff
@@ -24,9 +24,8 @@ void OnExit()
         [Test]
         public void SimpleTransition()
         {
-            var expected = "digraph {" + System.Environment.NewLine
```

Comment:
Have the expected results changed here, or is it just re-formatting? If the latter, could we revert the re-format so that it's clear things have stayed the same?